tools/eslint-rules/rules/no-ngrx-fail-action-without-error-action-implementation.spec.ts

```diff
@@ -10,8 +10,28 @@ const ruleTester = new TSESLint.RuleTester({
 });
 
 ruleTester.run(RULE_NAME, rule, {
-  // SPIKE TODO: ADD VALID ONES
-  valid: [],
+  valid: [
+    `        
+    import { ErrorAction } from '@spartacus/core';
+export class LoadProductFail implements ErrorAction {}
```

Comment:
[nitpick]
In valid cases, I believe we don't need this broken indentation. we can align it with the `import` statement above